tests/ImageSharp.Tests/Formats/Jpg/Block8x8FTests.cs

```diff
@@ -0,0 +1,402 @@
+﻿// Uncomment this to turn unit tests into benchmarks:
+//#define BENCHMARKING
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+using ImageSharp.Formats;
+using Xunit;
+using Xunit.Abstractions;
+// ReSharper disable InconsistentNaming
+
+namespace ImageSharp.Tests.Formats.Jpg
+{
+    public class Block8x8FTests : UtilityTestClassBase
+    {
+#if BENCHMARKING
+        public const int Times = 1000000;
```

Comment:
Good question. There are many arguments for and against.
The main issue with the benchmark project is, that currently it is very difficult to run dotTrace against them, while there is a straightforward way to profile unit tests. If performance is a long term goal, it's important to make profiling easy.